README.md

```diff
@@ -40,24 +40,29 @@ The docker compose file for this service is `docker-chs-development/services/tes
 ### Usage
 In order to use the generator, there are 2 possible endpoints that can be used.
 
-- POST: Sending a POST request on the following endpoint `{Base URL}/test-data/company)` will generate a new test company and accompanying Authcode. There is an optional parameter which is CompanySpec. When added to the request body will alter generated company to be based in Scotland or Northern Ireland but will default to England/Wales. An usage example looks like this: `{"jurisdiction":"scotland"}`
+- POST: Sending a POST request on the following endpoint `{Base URL}/test-data/company)` will generate a new test company and accompanying Authcode. There is an optional parameter which is CompanySpec. When added to the request body will alter generated company to be based in jurisdiction, company_status and type. The CompanySpec parameters are:
```

Comment:
`to be based in jurisdiction, company_status and type` does not sound good to me?